src/api/batch-import-export-api.ts

```diff
@@ -0,0 +1,31 @@
+import {PackageExportTransport} from "../interfaces/package-export-transport";
+import {httpClientV2} from "../services/http-client-service.v2";
+import {FatalError} from "../util/logger";
+
+class BatchImportExportApi {
+    public static readonly INSTANCE = new BatchImportExportApi();
```

Comment:
It feels to me like this is redundant, especially considering that the class is not exported

Reply:
This is the pattern that is used across the CLI. I think it is meant to provide singleton instances of classes.

Reply:
Sure, but you are still just exporting the instance created from the class (the class is not usable outside this file), and considering the the constructor is not private there is nothing preventing you from creating multiple instances of the class.
This pattern did not seem to be used in the service classes for instance